aad/client.bal

```diff
@@ -0,0 +1,423 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
If we are adding completely new file, we can change the year as 2021

Reply:
This was a new file. So changed the year thanks